README.md

```diff
@@ -1,8 +1,16 @@
 # Repository Template
 
-Start with an overview or a brief description of what the project is about and what it does.
+Start with an overview or a brief description of what the project is about and what it does. For example -
```

Comment:
Is it worth making a note in the main README about the git hooks and pre-commit?

Reply:
Thanks, a good suggestion. I created an [issue](https://github.com/nhs-england-tools/repository-template/issues/83) for this to be added to the `CONTRIBUTING.md` file as there is already a section that covers git hooks.